org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/services/XMLCompletions.java

```diff
@@ -350,6 +355,28 @@ private void collectOpenTagSuggestions(boolean hasOpenBracket, Range replaceRang
 		}
 	}
 
+	private void createPrologCompletionSuggestion(int tagNameEnd, String tag, CompletionRequest request,
+		CompletionResponse response) {

```

Comment:
Perhaps it's for an another PR, but IMHO I think we should have too completion on 

 * version attribute name
 * version attribute value (1.0, 2.0)
 * encoding  attribute name
 * encoding  attribute value (UTF-8, UTF-16, etc)

Reply:
I'll create an issue for that.